src/a11y/select/select.hooks.ts

```diff
@@ -45,9 +46,10 @@ export const useA11ySelect = <Option>(
     }
   });
 
-  useOnKey(buttonRef, ['Escape', 'Tab'], (event: KeyboardEvent) => {
+  useOnKey(optionListRef, ['Escape', 'Tab'], (event: KeyboardEvent) => {
```

Comment:
Why do you need to change this ref?

Reply:
The way we have the code it could still be the button, but if we take the list outside the button, we need to change the reference, and actually if I leave it like that it works correctly whether it is inside or outside the button, because it is actually a process that affects the list not the button.

Reply:
Oh nice, thank you. I think we can merge the PR